src/diagnostics/logPointsClient.ts

```diff
@@ -2,12 +2,15 @@ import * as req from "request";
 
 import * as child_process from 'child_process';
 import * as WebSiteModels from '../../node_modules/azure-arm-website/lib/models';
+// tslint:disable:max-classes-per-file
```

Comment:
Can you break up into multiple files vs just disabling the linter here?

Reply:
Will do

Reply:
Done.